store/schedule.go

```diff
@@ -321,7 +384,7 @@ func (s *Schedule) ValidateSchedule() []string {
 			errs = append(errs, er...)
 		}
 	} else {
-		if errStr := validateScheduleTime(s.ScheduleTime, s.AppId); errStr != "" {
+		if errStr := validateScheduleTime(s.ScheduleTime, app, conf.FutureScheduleCreationPeriod); errStr != "" {
```

Comment:
validateScheduleTime method accepts two arguments: int64 and string as the argument
Here you are passing three arguments: int64, App and string.
Is it correct?

Reply:
This method has been modified to handle the app-level configurations. It is working fine.